doc/tutorial/statistical_inference/putting_together.rst

```diff
@@ -11,16 +11,28 @@ Pipelining
 We have seen that some estimators can transform data and that some estimators
 can predict variables. We can also create combined estimators:
 
-.. image:: ../../auto_examples/compose/images/sphx_glr_plot_digits_pipe_001.png
-   :target: ../../auto_examples/compose/plot_digits_pipe.html
-   :scale: 65
-   :align: right
+.. raw :: html
+
+   <div class="sk-doc-div">
+    <div class="sk-doc-div-box">
 
 .. literalinclude:: ../../auto_examples/compose/plot_digits_pipe.py
     :lines: 23-63
 
+.. raw :: html
```

Comment:
![Screenshot from 2020-07-14 15-05-57](https://user-images.githubusercontent.com/1663864/87429072-90742400-c5e3-11ea-81b0-319c9d6cc21d.png)
this is putting the plot on the side and there isn't enough space for the code beside it. Could we put the image after the code block?

Reply:
Ok, you convinced me for this one: the codeblock is too high and the scrollbar is not accessible at the beginning of the block. Thanks.